source/_posts/2020-06-24-release-112.markdown

```diff
@@ -0,0 +1,1274 @@
+---
+layout: post
+title: "0.112: Beta release notes"
+description: "0.112: Beta release notes"
+date: 2020-06-24 00:00:00
+date_formatted: "July 1, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-06-0.112/social.png
+---
+
+<a href='/integrations/#version/0.112'><img src='/images/blog/2020-06-0.112/social.png' style='border: 0;box-shadow: none;'></a>
+
+And another 3 week passed! Home Assistant Core 0.112.0!
+
+Please note, these are the beta release notes and might still change during
+the beta week.
+
+## Important upgrade notice!
+
+Let's start with an important notice for upgrading to Home Assistant Core 0.112.
+
+This release has changes to the database format of Home Assistant. On upgrade,
```

Comment:
This would also indicate that rolling back to an earlier version could be difficult / impossible. Should we warn for this?